src/ophyd_async/epics/motion/motor.py

```diff
@@ -14,33 +14,39 @@ class Motor(StandardReadable, Movable, Stoppable):
 
     def __init__(self, prefix: str, name="") -> None:
         # Define some signals
-        self.setpoint = epics_signal_rw(float, prefix + ".VAL")
-        self.readback = epics_signal_r(float, prefix + ".RBV")
+        self.user_setpoint = epics_signal_rw(float, prefix + ".VAL")
+        self.user_readback = epics_signal_r(float, prefix + ".RBV")
         self.velocity = epics_signal_rw(float, prefix + ".VELO")
-        self.units = epics_signal_r(str, prefix + ".EGU")
+        self.max_velocity = epics_signal_r(float, prefix + ".VMAX")
+        self.acceleration = epics_signal_rw(float, prefix + ".ACCL")
+        self.motor_egu = epics_signal_r(str, prefix + ".EGU")
         self.precision = epics_signal_r(int, prefix + ".PREC")
-        # Signals that collide with standard methods should have a trailing underscore
-        self.stop_ = epics_signal_x(prefix + ".STOP")
+        self.max_resolution = epics_signal_r(float, prefix + ".MRES")
```

Comment:
Calling `MRES` "max resolution" seems lisleading - I'd rather call it `motor_step_size`. The resolution of the encoder position readback might be higher. 

Do we also want `ERES` for `encoder_step_size`?

Reply:
My bad, was a poor copy/paste, thanks.

I'm keen not to pre-emptively put in too much before we know we need it and Hyperion has yet to need `ERES`. I guess we could if we were trying to be more clever about tolerances on how close we are to things but we're not currently...

Reply:
`ERES` defaults to `MRES`, so is actually the right thing to look at to work out encoder resolutions, `RDBD` is better to look for tolerances, as the motor record will treat this value as "how close to the readback do I need to be to be done", and I think it will ignore move requests within this deadband...

Reply:
Is `RDBD` guaranteed to be set to non-zero?

Reply:
No idea about guarantee, but a spot check of 3 motors showed that it seems to default to `MRES` unless overridden to be bigger

Reply:
Ok, I've called it `tolerance` rather than `RDBD` as I think this makes more sense to a user of the device but happy to be argued with

Reply:
Can we make it `deadband` instead? Sounds like tolerance to me, and closer to the EPICS name...